letsencrypt-auto

```diff
@@ -14,7 +14,12 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if ! type "sudo" > /dev/null; then
+    su
```

Comment:
This will open new shell session, none of the commands following it would be executed in that session - in act none of the following commands would be executed :(